FAQ.md

```diff
@@ -1,19 +1,37 @@
-# 🔮 FAQ
-
-1. ## Fonts don't seem to be correct
-
-   Add `@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:300:400,600&display=swap');` to your applications CSS.
-
-2. ## Content-Security-Policy (CSP) problems?
-
-   You need to configure DEV env to allow
-
-   ```bash
-   style-src: 'unsafe-inline'
-   ```
-
-   And also might need to set `dangerously omit style nonce` to `true`.
-
-3. ## Some components won't work on old browsers (e.g. IE) at all
-
-   Add polyfills to your application.
+# 🔮 FAQ
+
+## 1. Fonts don't seem to be correct
+
+Add `@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:300:400,600&display=swap');` to your applications CSS.
+
+## 2. Content-Security-Policy (CSP) problems?
+
+If strict content security policy is not required, you can simply use
+
+```bash
+style-src: 'unsafe-inline'
+```
+
+to allow the use of dynamic inline styling.
+
+### Strict content security policies
+
+Strict content security policy does not allow dynamic styling without extra safety measures. Styled components uses `__webpack_nonce__` to provide these measures.
+
+Add `__webpack_nonce__ = '<some generated nonce>'` in the beginning of your entry file. It must be defined before any other scripts. One way to do this is to introduce the nonce as above in a new file and import that in your entry file. More information can be found in [webpack documentation.](https://webpack.js.org/guides/csp/)

```

Comment:
'<some generated nonce>' => maybe would be better to say "some generated uuid" or somehow point out already the nature of nonce.

Reply:
Nonce itself already is both a regular English word for something intended for single-use, so I wouldn't introduce another one for that. I'll try to explain it somehow since it's not exactly the most common term to be used.